homeassistant/components/climate/ecobee.py

```diff
@@ -195,9 +195,12 @@ def is_away_mode_on(self):
         mode = self.mode
         events = self.thermostat['events']
         for event in events:
-            if event['running']:
+            if event['running'] and event['holdClimateRef'] == 'away':
```

Comment:
Since both if statements result in `mode` being set to `away`, please merge both into 1 statement.